### ISSUE TITLE
Select a default theme depending on the operating system's theme

### ISSUE THREAD

**Issue by peepo5** [tenacityteam/tenacity-legacy#178]
- [x] I have read the specified guidelines for issues

**Is your feature request related to a problem? Please describe.**
I am frustrated when new users use light theme and do not know there is a dark theme (or will be #83 )

**Describe the solution you'd like**
I would like dark theme to be the default

**Describe alternatives you've considered**
A popup when installed asking which theme the user prefers.

**Comment by caughtquick** [comment]
Duplicate #29, #83

**Comment by edo0** [comment]
Given the historical legacy and professional reputation of Audacity which Tenacity aims to carry forward, I personally deem following the latest fad of dark themes is a mistake which should be avoided.
I am favourable to having an option in the preferences menu, but I do not think the default implementation of a dark theme would be a right choice

**Comment by peepo5** [comment]
It is not a duplicate, this is different issue around defaulting

**Comment by ErikPrantare** [comment]
I don't think there's a reason to switch the default to dark theme, but making it match the system theme would be an improvement imo (if it doesn't already do this)

**Comment by peepo5** [comment]
theme should be system i agree, but i still think dark would be baseder

**Comment by caughtquick** [comment]
> theme should be system i agree, but i still think dark would be baseder

I think it should go to system theme, then if that's not set go to a light theme. I don't see any point in going to a dark theme by default as most people don't use dark theme.

**Comment by BeetMacol** [comment]
Having a nice first time use configuration popup is a good solution for that imo. I'm seeing more and more apps doing that and it would be great for Tenacity too. That popup could contain both the color palette and the icons on separate configuration pages. It could also be used for some non-visual (not theme related) settings that are frequently changed by users.

**Comment by peepo5** [comment]
Yeah like most changed config options

**Comment by NotAProton** [comment]
[ardour](https://ardour.org/)
[audition](https://helpx.adobe.com/audition/user-guide.html/premiere-pro/how-to/essential-sound-panel.ug.html)
Dark mode is sort of a industry standard. I say we not only make dark mode the default, we, eventually, deprecate light mode entirely.

**Comment by unbeatable-101** [comment]
Nah, unless most major operating systems deprecate lightmode, we should keep both to keep the transition from being jarring to people using lightmode

**Comment by Songtech-0912** [comment]
I understand the historical precedence of having light theme as default. If dark theme seems aesthetically more pleasing or slightly less harsh on the eyes, we can just adjust the light theme to use a slightly warmer and less bright shade of whitish gray.

**Comment by to7m** [comment]
Dark mode is not a fad; it is nicer to look at and is becoming increasingly more standard.

Light mode with less brightness would be worse in my opinion, because without the contrast, the screen backlight itself would need to be brighter than otherwise. With some software (word processors, notation software), this is a valid approach because people expect to see a background resembling white paper, but I personally don't see the point when it comes to DAWs.

All irrelevant if Tenacity can just blend properly with the system theme though :)

**Comment by j127** [comment]
Dark mode is hard to read for some people, especially with certain lighting. See [here](https://zapier.com/blog/dark-mode-bad-productivity/), [here](https://tidbits.com/2019/05/31/the-dark-side-of-dark-mode/), and [here](https://www.techradar.com/news/the-dark-mode-craze-may-do-more-harm-than-good-this-is-why).

**Comment by n0toose** [comment]
> Dark mode is hard to read for some people, especially with certain lighting.

This is correct. Had a friend of mine with astigmatism talk to me about this when I was working on another project.

**Comment by GentlemansKiller** [comment]
Why the arguing. Just make a dark theme and let the user decide which one they want to be default... Or make a toolkit where ppl can create and export themes and import and make those default/nondefault

ez

**Comment by caughtquick** [comment]
> Why the arguing. Just make a dark theme and let the user decide which one they want to be default... Or make a toolkit where ppl can create and export themes and import and make those default/nondefault
> 
> 
> 
> ez

Yep, let's go for the system theme and maybe a prompt when you startup.

**Comment by n0toose** [comment]
See https://github.com/tenacityteam/tenacity/issues/83#issuecomment-877481788.

**Comment by unfa** [comment]
I prefer dark themes as they help me focus on the content I am working on, without blinding me needlessly.
Everything on my system is set to a dark theme because of that.

I think it'd make sense if Tenacity followed system settings to respect user's preference.
